tools/test-helpers/helpers.js

```diff
@@ -0,0 +1,35 @@
+window.D2L = window.D2L || {};
+window.D2L.TestHelpers = window.D2L.TestHelpers || {};
+
+export async function focus(selector, options) {
+	const elemToFocus = queryShadowSelector(selector);
+	if (!options || !options.waitFor) {
+		elemToFocus.focus();
+		return;
+	}
+	return new Promise((resolve) => {
+		const elemToWait = queryShadowSelector(options.waitFor.selector);
+		elemToWait.addEventListener(options.waitFor.eventName, resolve, { once: true });
```

Comment:
Should we be removing the event listener?

Reply:
The `{ once: true }` option eliminates the need to remove.  Although, it is not supported by IE11, so if we want to use this helper there, then I should switch it to the old fashioned way.

Reply:
Oh nice!